.github/workflows/python-package.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
     - name: Install dependencies
       run: |
-        python -m pip install coveralls 'tox<4.0.0' tox-factor
+        python -m pip install coveralls 'tox<4.0.0' tox-factor setuptools
```

Comment:
`setuptools` used to be automatically installed in virtual environments but this was removed in Python 3.12 (https://github.com/python/cpython/issues/95299)